src/style.css

```diff
@@ -1,3 +1,81 @@
+/* Reset default browser styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+ul,
+li {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  --blue: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  background-color: #eee;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  background-color: #fff;
+}
+
+.logo {
+  display: block;
+  padding-top: 10px;
+  align-items: right;
```

Comment:
The property `align-items: right;` is not valid in CSS. If you intend to align the logo to the right, consider using `text-align: right;` on a parent element or adjust the flex properties appropriately.